dotcom-rendering/src/amp/components/ContentABTest.tsx

```diff
@@ -26,7 +26,9 @@ export const getGroup = (pageId: string): ContentABTestGroup => {
 	// Add the leading slash as this will be present when we apply the equivalent algorithm at the analysis stage
 	const hashedPageId = sha256(`/${pageId}`);
 	// Take the last 4 bytes of the hash
-	const lastFourBytes = hashedPageId.words[hashedPageId.words.length - 1];
+	const [lastFourBytes] = hashedPageId.words.slice(-1);
+	if (!lastFourBytes) throw new Error('Missing lastFourBytes');
```

Comment:
Do we want to fail in a softer way?

Reply:
I think due to how `sha256` function works it should always return a non-empty word array. Nevertheless I think we can just default to `NaN` if lastFourBytes is missing and delegate to the later error handling already present in this function.